bash-preexec.sh

```diff
@@ -199,14 +199,16 @@ __bp_preexec_invoke_exec() {
         if type -t "$preexec_function" 1>/dev/null; then
             __bp_set_ret_value $__bp_last_ret_value
             $preexec_function "$this_command"
-            preexec_ret_value="$?"
+            if (( $? != 0 )); then
```

Comment:
LGTM. This preserves the "blocking" behavior of preexec returning non-zero even if another preexec function returns 0, which is ideal.